src/clean.cmd.js

```diff
@@ -54,8 +54,10 @@ class CleanCommand {
       this._cacheDir = path.resolve(this._cwd, '.hlx', 'cache');
     }
 
-    await clean(this._target);
-    await clean(this._cacheDir);
+    await Promise.all([
```

Comment:
why this change? unrelated to logger :-)

Reply:
Yes, it's unrelated, but: `await await` enforces the sequence, `Promise.all` starts the processes in parallel and allows completion in any sequence.